frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css

```diff
@@ -24,7 +24,8 @@ body {
   background-color: var(--mb-color-bg-dashboard);
 }
 
-[data-metabase-theme="night"][data-metabase-theme="night"] {
+[data-metabase-theme="night"][data-metabase-theme="night"],
+[role="tooltip"] {
```

Comment:
not only the dark theme, but all our tooltips have dark background so we need to style it like dark theme.